src/env/index.ts

```diff
@@ -12,8 +21,3 @@ export const getEnv = () => env || getBrowserEnv();
 export const getDocument = (): Document => getEnv().document;
 
 export const getWindow = (): Window => getEnv().window;
-
-export const setEnvForTests = (window: Window) => {
-  env.document = window.document;
-  env.window = window;
-};
```

Comment:
this was specific for tests, for now setEnv is a necessity of delaying an import / definition issue, and i don't think it is a feature yet. I would like this to stay setEnvForTests ( and even that was some kind of issue of testing and not a real need ) and setEnv to not be exposed

Reply:
I was thinking we should expose `setEnv` for devs who work on a strange env, someone wrote about it in #8208

Reply:
in case we will have specific pr with new feature for it after evaluating which are the side effects.